packages/pay/README.md

```diff
@@ -218,43 +219,43 @@ If the connection was established, notify receiver to close the connection. For
+| :----------------------------------------- | :---------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`destinationAsset`**                     | **[`AssetDetails`](#assetdetails)**                                                                                     | Destination asset and denomination, resolved using Open Payments or STREAM, or provided directly.                                                                                                                                                               |
+| **`destinationAddress`**                   | `string`                                                                                                                | ILP address of the destination STREAM recipient, uniquely identifying this connection.                                                                                                                                                                          |
+| **`sharedSecret`**                         | `Uint8Array`                                                                                                            | 32-byte seed to derive keys to encrypt STREAM messages and generate ILP packet fulfillments.                                                                                                                                                                    |
+| **`receivingPaymentDetails`** (_Optional_) | **[`IncomingPayment`](#incoming-payment)**                                                                              | Open Payments Incoming Payment metadata, if the payment pays into an Incoming Payment.                                                                                                                                                                          |
+| **`accountUrl`** (_Optional_)              | `string`                                                                                                                | URL of the recipient Open Payments/SPSP account (with well-known path, and stripped trailing slash). Each payment pointer and its corresponding account URL identifies a unique payment recipient. Not applicable if STREAM credentials were provided directly. |
+| **`receivingAccount`** (_Optional_)        | `string`                                                                                                                | Payment pointer, prefixed with "\$", corresponding to the recipient Open Payments/SPSP account. Each payment pointer and its corresponding account URL identifies a unique payment recipient. Not applicable if STREAM credentials were provided directly.      |
```

Comment:
:thinking: 
Maybe we need to decide if @interledger/pay's [description](https://github.com/interledgerjs/interledgerjs/tree/master/packages/pay#pay-money_with_wings) should be changed to:
>Send Open Payments Outgoing Payments over Interledger using STREAM

If so, I think it makes sense to match the Outgoing Payments fields in the Open Payments spec as much as possible. Maybe this ends up looking like the outgoing payment lifecycle methods in Rafiki, where an outgoing payment is passed to and "returned" from each method?

Otherwise, if we still consider Open Payments-free stream payments to payment pointers a supported use case, this PR could arguably be limited to its original scope of just switching from invoices to Incoming Payments.

(I think my review comments have been leading us towards the former.)

Reply:
@wilsonianb are you suggesting these changes for Open Payments, for ilp-pay, or for both?

Reply:
I'm suggesting keeping `destination*` throughout ilp-pay (minimizing changes in this pr like you originally intended).
And maybe we also change the Open Payments spec to use `destintination*`
https://docs.openpayments.guide/discuss/622b6e7fc2751c00146958c3